collaborative_practice.js

```diff
@@ -32,6 +37,21 @@ const runProgram = async () => {
                 case "1":
                     listUsers(users);
                     break;
+
+                case "2":
+                    const newUser={
+                        id:users.length+1,
+                        firstName:await prompt("FirstName:"),
+                        lastName:await prompt("Lastname:")
```

Comment:
Please read the two inputs in separate variables
Don't do more than one thing at a time!